sterror/sterror.go

```diff
@@ -0,0 +1,157 @@
+package sterror
+
+import (
+	"errors"
+)
+
+type Op string
+
+func (o Op) String() string {
+	return string(o)
+}
+
+type Scope string
+
+func (s Scope) String() string {
+	return string(s)
+}
+
+type Info string
+
+func (i Info) String() string {
+	return string(i)
+}
+
+const (
+	colon     = ": "
+	hyphen    = " - "
+	separator = "\n"
```

Comment:
Why not 'newline'?

Reply:
What is the difference? Is 'newline' the same?

Reply:
You named the colon char "colon", the hyphen char "hyphen" and the newline char "separator". Seems inconsistent.
They could all be separators somehow.